ml_garden/core/steps/calculate_metrics.py

```diff
@@ -19,8 +29,8 @@ def __init__(self) -> None:
         super().__init__()
         self.init_logger()
 
-    def _calculate_metrics(self, true_values: pd.Series, predictions: pd.Series) -> dict:
-        """Calculate metrics.
+    def _calculate_regression_metrics(self, true_values: pd.Series, predictions: pd.Series) -> dict:
```

Comment:
It _might_ be a good idea to have some `_calculate_metrics` that takes `task: SupportedTasks` and calls the appropriate method. This allows users to be more programmatic with these calls, though I'm not sure it's a usecase ML Garden users would face at all.